### DISCUSSION TITLE
Z-wave JS Conroller showing "Initialising network" z-wave node showing "Node not ready"

### DISCUSSION THREAD

**Discussion by widert** [zwave-js/node-red-contrib-zwave-js#268] [Q&A] [answered]
Just trying to add another node to my z-wave network, but the inclusion seems to have messed up one of my existing nodes... :(

The node seems now to be asleep (it should been awake)
![Capture1](https://user-images.githubusercontent.com/29115503/226139731-91376dec-c33e-43ba-b7ce-24dd24945f83.JPG)

and when clicking on the node I get "Node not ready":
![image](https://user-images.githubusercontent.com/29115503/226139876-af6e5bcb-da3a-4fbf-936f-f66b90866c96.png)

Anything I can do to fix it again? Or is exclude and include again by best option?

**Comment by marcus-j-davies** [comment, marked as answer]
HI @widert 

You can try waking up the node - if it can sleep, most devices have a wake routine, it should be documented in its manual.
Once the driver receives a wake up command from it - it will re-interview the node, and put it back to an operating state.

If for what ever reason, the node is not waking up, you can shift + click the node - and this will reveal options, one of which is a "remove failed node" option - once removed, you will need to re-add it, but I would first try the wake up routine, before having to forcibly remove and re-add it

**Reply by marcus-j-davies** [reply]
See here for shift + click

https://github.com/zwave-js/node-red-contrib-zwave-js/wiki/User-Interface

**Comment by widert** [comment]
Thanks Marcus, the node was an "old" one that reappeared, removed it with Shift-Click

I also used the opportunity to upgrade to the newest version, looks great, congratulation with good work!

Regards,
Tom
